test/truffle/docker/oraclelinux/Dockerfile

```diff
@@ -21,9 +24,15 @@ USER test
 ENV GRAALVM_VERSION=0.24
 ADD graalvm-$GRAALVM_VERSION-linux-amd64-jdk8.tar.gz /test
 ENV PATH=/test/graalvm-$GRAALVM_VERSION/bin:$PATH
+ENV RUBY_HOME=/test/graalvm-$GRAALVM_VERSION/language/ruby
+
+# Build the AOT image
+RUN aot-image -H:Name=aot-ruby --ruby
+ENV PATH=/test:$PATH
 
 # Straight away, just run the ruby executable as the most basic test
 RUN ruby --version
+RUN aot-ruby -Xhome=$RUBY_HOME --version
```

Comment:
> I made it that location under the assumption you'd `cd` into the root

This is a continuation of the discussion we had about how different people worked with tar balls - I'd never think to `cd` into the root.

I'll try some alternatives and do a PR.